src/main/kotlin/nl/info/client/zgw/ztc/ZtcClientService.kt

```diff
@@ -362,6 +367,21 @@ class ZtcClientService @Inject constructor(
             ztcClient.informatieObjectTypeRead(informatieobjecttypeUUID)
         }
 
+    fun readEigenschappen(zaaktype: URI) =
+        uriToEigenschappenCache.get(zaaktype) {
+            val response = ztcClient.eigenschapList(
+                EigenschapListParameters().apply {
+                    this.zaaktype = zaaktype
+                    status = EigenschapListParametersStatus.DEFINITIEF
+                }
+            )
+            response.results
+        }
+
+    fun readEigenschap(zaaktype: URI, eigenschap: String) =
+        this.readEigenschappen(zaaktype).find { it.naam == eigenschap }
+            ?: throw EigenschapNotFoundException("No eigenschap with naam '$eigenschap' found for zaaktype '$zaaktype'.")
```

Comment:
Strictly speaking: and with EigenschapListParametersStatus.DEFINITIEF ? Maybe a bit too much to add here?